examples/06_scope_functions/01_let.md

```diff
@@ -0,0 +1,38 @@
+# let
+
+`let` is a Kotlin standard library function that can be used for scoping and null-checks. When called on an object, `let` executes the given functional block and returns the result of its last expression.
+The object is accessible inside the block by the reference `it`. 
+
+<div class="language-kotlin" theme="idea" data-min-compiler-version="1.3">
+
+```kotlin
+var i = 0
+fun funWithSideEffect(): String = ".".repeat(i++)
+
+fun main() {
+//sampleStart
+    val r = funWithSideEffect().let {                              // 1
```

Comment:
I guess the variable should have the normal name instead of one "random" letter

Reply:
Rewrote the whole example